legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/h2Extension.pure

```diff
@@ -157,6 +157,14 @@ function <<access.private>> meta::relational::functions::sqlQueryToString::h2::c
       'cast( parseDateTime('+$params->at(0)+','+$dateFormat +') as date)';
 }
 
+function <<access.private>> meta::relational::functions::sqlQueryToString::h2::transformParseDateParams(params:String[*]):String[2]
```

Comment:
It seems that you want to make the format an optional parameter. Original behavior remains same. Correct? What is the purpose of this change?